platform/microprofile/README.md

```diff
@@ -0,0 +1,7 @@
+# Trellis Deployable Application
```

Comment:
Ok, I'm being pushy here, but can we explicitly mention that this artifact uses the triplestore backend? I'm thinking of the casual browser who is just trying to figure out what different modules do.

Reply:
Yes, I'll fill that out some more.